music21/harmony.py

```diff
@@ -113,6 +113,9 @@
     ('Tristan', ['1,#4,#6,#9', ['tristan']]),  # Y
 ])
 
+VALID_ABBRS = [abbr for chord, (notes, abbrs) 
+                    in CHORD_TYPES.items() for abbr in abbrs]
+
```

Comment:
This logic is obscure to me.  abbr is being used for two different variables at the same time.  nested comprehensions are something I avoid.  please rewrite as for loops.